test/utils/session-core.test.js

```diff
@@ -39,7 +39,7 @@ describe('setDocContainerAclPermission', () => {
     expect(sessionHelpers.setDocAclForUser).toBeCalledWith(
       session,
       expectedContainerUrl,
-      'update',
+      '',
```

Comment:
Changed this to '' to better communicate that this value doesn't do anything. The only value that matters here is "create." All other strings have the "update" effect.